### PR TITLE
Feedback round 5

### PR DESCRIPTION
**Pull request by Kwonunn**
Implements #195, #196, #197 and #219.

### CODE REVIEW

File: source/blender/io/ply/exporter/ply_file_buffer_ascii.hh [hunk 1]
```diff
@@ -27,44 +27,18 @@ class FileBufferAscii : public FileBuffer {
  public:
   using FileBuffer::FileBuffer;
```
Comment (Stroopwafe1) [inline source/blender/io/ply/exporter/ply_file_buffer_ascii.hh:27]: This using has to go above the public declaration according to the style guide

File: source/blender/io/ply/exporter/ply_file_buffer_binary.hh [hunk 2]
```diff
@@ -29,63 +29,18 @@ class FileBufferBinary : public FileBuffer {
  public:
   using FileBuffer::FileBuffer;
```
Comment (Stroopwafe1) [inline source/blender/io/ply/exporter/ply_file_buffer_binary.hh:29]: Same as ascii, using declaration above public

**Review by Stroopwafe1 — COMMENTED** [review]
In general: Take a close look at the style guide for classes and make sure the filebuffers are following the style guide. This way Hoogly has nothing to comment about anymore
https://wiki.blender.org/wiki/Style_Guide/C_Cpp#Class_Layout

**Review by Nathan-Rozendaal — CHANGES_REQUESTED** [review]
Unit tests fail (they don't fail on development)

**Review by Nathan-Rozendaal — APPROVED** [review]

**Pull request merged**